yarn-project/simulator/src/public/avm/fixtures/simple_contract_data_source.ts

```diff
@@ -22,6 +22,8 @@ export class SimpleContractDataSource implements ContractDataSource {
   private contractInstances: Map<string, ContractInstanceWithAddress> = new Map();
   // maps contract instance address to address
   private contractArtifacts: Map<string, ContractArtifact> = new Map();
+  // maps contract class ID & function selector to name
+  private debugFunctionName: Map<string, Map<string, string>> = new Map();
```

Comment:
Optional: you might be able to simplify to `Map<string,string>` if you just concatenate the class id and function selector like `id:selector`.